flink-end-to-end-tests/test-scripts/test-runner-common.sh

```diff
@@ -39,32 +43,51 @@ function run_test {
     export TEST_DATA_DIR=$TEST_INFRA_DIR/temp-test-directory-$(date +%S%N)
     echo "TEST_DATA_DIR: $TEST_DATA_DIR"
 
+    export DESCRIPTION="$description"
```

Comment:
we could explicitly pass these to `post_test_validation` as we did before.

Reply:
Thanks, I thought it was not working before. Will undo `description` and `check exception` then.